c2corg_ui/static/js/card.js

```diff
@@ -107,4 +107,94 @@ app.CardController.prototype.createURL = function() {
 };
 
 
+/**
+ * Gets the global ratings for each activity of a route.
+ * @export
+ * @return {Object} rating
+ */
+app.CardController.prototype.getGlobalRatings = function() {
+  var doc = this.doc;
+  var rating = {};
+
+  for (var i = 0; i < doc.activities.length; i++) {
+    var a = doc.activities[i];
+    if (a === 'rock_climbing' || a === 'mountain_climbing' || a === 'snow_ice_mixed') {
```

Comment:
maybe more elegant, but apparently no one on Stackoverflow is recommending using a conditional statements within a switch statement. In my case, I have a boolean to evaluate.

Or if needed, Stackoverflow suggests doing something ugly like 
```

switch (true) {
    case liCount == 0:
        setLayoutState('start');
        var api = $('#UploadList').data('jsp');
        api.reinitialise();
        break;
    case liCount<=5 && liCount>0:
```